server/modules/user/api/userApi.js

```diff
@@ -149,10 +151,16 @@ export const init = app => {
   })
 
   const _updateUser = async (req, res) => {
-    const userToUpdate = Request.getBody(req)
-
+    const body = Request.getBody(req)
+    const { user: userToUpdateString } = body
+    let userToUpdate = A.parse(userToUpdateString)
     const validation = await UserValidator.validateUser(userToUpdate)
 
+    userToUpdate = {
+      ...userToUpdate,
+      props: A.stringify(userToUpdate.props),
+    }
```

Comment:
I am worried about that because due to we use formData, we should do this

Reply:
what?

Reply:
The repository expects that the props are a string, maybe it is better to move this logic to stringify to the repository.

Reply:
can u make it the same way we persist props for nodeDefs ?
This code might be old... better we fix it now.
what do you think?

Reply:
Perfect! I will take a look into the nodeDef code, yep I am finding code that is not used and outdated.